docs/releasing/publishing.md

```diff
@@ -1,17 +1,35 @@
 # Publishing GOV.UK Frontend
 
-1. Make sure you have cards in the "Backlog" column to:
-  - Update the GOV.UK Design System to use the latest release
-  - Update the GOV.UK Prototype Kit to use the latest release
-  - Update the GOV.UK Frontend Docs to use the latest release
+## Steps you must complete before you release a new version of GOV.UK Frontend
 
-2. Checkout **master** and pull latest changes.
+1. Raise new issues in the team GitHub repositories to:
+- Create release notes for the new release, for example, [#1986](https://github.com/alphagov/govuk-frontend/issues/1986)
+- Create release notes for the new GOV.UK Prototype Kit v[VERSION NUMBER]   
+- Release v[VERSION NUMBER], for example, [#1987](https://github.com/alphagov/govuk-frontend/issues/1987)
+- Update the GOV.UK Design System to use the new release, for example, [#1347](https://github.com/alphagov/govuk-design-system/issues/1347)
+- Update the GOV.UK Frontend Docs to use the new release, for example, [#75](https://github.com/alphagov/govuk-frontend-docs/issues/75)
+- Update the GOV.UK Prototype Kit to use the new release, for example, [#923](https://github.com/alphagov/govuk-prototype-kit/issues/923)
+- Release GOV.UK Prototype Kit v[version number], for example, [#917](https://github.com/alphagov/govuk-prototype-kit/issues/917)
 
-3. Run `nvm use` to ensure you are using the right version of Node.js and npm.
+2. Add the issues to the **Design System Sprint Board**.
 
-4. Run `npm install` to ensure you have the latest dependencies installed.
+3. Confirm that any related content for the Design System or Frontend docs is ready to publish.
 
-5. Create and checkout a new branch (`release-[version-number]`).
+4. Write a summary of the release. For example:
```

Comment:
Do we want to explicitly say that this summary is for Slack updates, otherwise it may be unclear why we're writing release notes and a summary.

Reply:
Really good idea.

Reply:
Thanks both 👍